docs/rules/no-test-module-for.md

```diff
@@ -0,0 +1,23 @@
+# Use `module` instead of `moduleFor` (no-test-module-for)
+
+`moduleForComponent`, `moduleFor`, `moduleForAcceptance`, etc have been deprecated and there are codemods to help migrate.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+moduleFor('Test Name');
+```
+
+Examples of **correct** code for this rule:
+
+```js
+module('Test Name', function(hooks) {
+
+});
+```
+
+## Migration
```

Comment:
Can you add a `## References` section with links to spec for these functions and the deprecation notice?

There's a documentation template here: https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/_TEMPLATE_.md

Reply:
maybe addressed here? 96fc16ebc6b1958317e5745fca894833070ef968